db/seeds.rb

```diff
@@ -11,53 +11,90 @@
 require 'json'
 require 'faker'
 
+# Purge contents of 'UserItems' table
+unless UserItem.all.empty?
+  puts "Purging 'UserItems' table..."
+  UserItem.all.each do |user_item|
+    UserItem.destroy(user_item.id)
+    # puts "  Records remaining: #{UserItem.count}"
+  end
+end
+
 # Purge contents of 'Users' table
 unless User.all.empty?
   puts "Purging 'Users' table..."
   User.all.each do |user|
     User.destroy(user.id)
-    puts "  Records remaining: #{User.count}"
+    # puts "  Records remaining: #{User.count}"
   end
 end
 
-# Seed 'Users' table
-puts 'Creating users...'
-3.times do
-  puts "  Created #{User.create!(email: Faker::Internet.email, password: '123456').email}"
+# Purge contents of 'StoreItems' table
+unless StoreItem.all.empty?
+  puts "Purging 'StoreItems' table..."
+  StoreItem.all.each do |store_item|
+    StoreItem.destroy(store_item.id)
+    # puts "  Records remaining: #{StoreItem.count}"
+  end
 end
-puts "Seeding users complete!\n\n"
 
 # Purge contents of 'Stores' table
 unless Store.all.empty?
   puts "Purging 'Stores' table..."
   Store.all.each do |store|
     Store.destroy(store.id)
-    puts "  Records remaining: #{Store.count}"
+    # puts "  Records remaining: #{Store.count}"
   end
 end
 
-# Seed 'Stores' table
-puts 'Creating stores...'
-5.times do
-  puts "  Created #{Store.create!(
-    name: Faker::Commerce.vendor,
-    address: Faker::Address.full_address,
-    image_url: Faker::Internet.url,
-    longitude: Faker::Address.longitude,
-    latitude: Faker::Address.latitude
-  ).name}"
-end
-puts "Seeding stores complete!\n\n"
-
 # Purge contents of 'Items' table
 unless Item.all.empty?
   puts "Purging 'Items' table..."
   Item.all.each do |item|
     Item.destroy(item.id)
-    puts "  Records remaining: #{Item.count}"
+    # puts "  Records remaining: #{Item.count}"
   end
 end
 
+# Seed 'Users' table
+puts "\nCreating users..."
+3.times do
+  puts "  Created #{User.create!(email: Faker::Internet.email, password: '123456').email}"
+end
+puts "\nSeeding users complete!\n\n"
+
+# Seed 'Stores' table
+puts 'Creating stores...'
+puts "  Created #{Store.create!(
+  name: 'IGA',
+  address: Faker::Address.full_address,
+  image_url: Faker::Internet.url,
+  longitude: Faker::Address.longitude,
+  latitude: Faker::Address.latitude
+).name}"
+puts "  Created #{Store.create!(
+  name: 'Metro',
+  address: Faker::Address.full_address,
+  image_url: Faker::Internet.url,
+  longitude: Faker::Address.longitude,
+  latitude: Faker::Address.latitude
+).name}"
+puts "  Created #{Store.create!(
+  name: 'PA',
+  address: Faker::Address.full_address,
+  image_url: Faker::Internet.url,
+  longitude: Faker::Address.longitude,
+  latitude: Faker::Address.latitude
+).name}"
+puts "  Created #{Store.create!(
+  name: 'Provigo',
+  address: Faker::Address.full_address,
+  image_url: Faker::Internet.url,
+  longitude: Faker::Address.longitude,
+  latitude: Faker::Address.latitude
+).name}"
```

Comment:
for the store addresses it would be beneficial to put Montreal addresses. Not sure where we are with the [geocoder](https://github.com/alexreisner/geocoder?tab=readme-ov-file#basic-search) gem implementation, but it can be useful here to generate the latitude and longitude from the address, like so:
```
results = Geocoder.search("Paris")
results.first.coordinates
# => [48.856614, 2.3522219]  # latitude and longitude
```

Reply:
@gabrieleli09 is working on the geocoder.
@sofofos Would it make sense to use OpenMap APIs to provide the coordinates of the supermarket locations? I'd have to read their documentation on how to do so though.

Reply:
@sofofos I manually added full addresses and coordinates to each of the stores via PR#29.